projects/sensory_interface/public/app.js

```diff
@@ -131,4 +146,39 @@ function onCellChange(event) {
     let selectedValue = elementUnderTouch.firstChild.data;
     startSoundPlayback(selectedValue);
     event.stopPropagation();
+}
+
+function onRadioChange(radio) {
+    let minValuePicker = document.getElementById("minValue");
```

Comment:
If values are already set, don't override them.
Example scenario:
User chooses manual, changes the values, then auto, then wants to go back to manual, but the values were overridden.